src/MeasureToolbox/integrals.jl

```diff
@@ -239,6 +325,43 @@ function generate_integral_data(pref::InfiniteOpt.GeneralVariableRef,
                                   weight_func = weight_func)
 end
 
+#Function for generating coefficients for FE Gauss Lobatto Quadrature
+function _lobatto_coeff(points::Array{Float64}, num_nodes::Int = 10)
+    sort!(points)
+    (_, coeffs) = FastGaussQuadrature.gausslobatto(num_nodes)
+    coeff_vector = zeros((length(points) - 1)*(num_nodes - 1) + 1)
+    idx = 1:num_nodes
+    for i in 1:length(points)-1
+        coeffs_modified = ((points[i+1] - points[i]) / 2 * coeffs)
+        coeff_vector[idx] += coeffs_modified
+        idx = idx .+ (num_nodes - 1)
+    end
+    return coeff_vector
+end
+
+#Single Pref Finite Gauss Lobatto Quadrature
+function generate_integral_data(pref::InfiniteOpt.GeneralVariableRef,
+        lower_bound::Real,
+        upper_bound::Real,
+        method::FEGaussLobatto;
+        num_nodes::Int = 3,
+        weight_func::Function = InfiniteOpt.default_weight,
+        kwargs...)
+    if lower_bound == -Inf || upper_bound == Inf
+        @warn("The Gauss Lobatto method can only be applied on finite intervals, " *
+              "switching to an appropriate method.")
+    end
+    if num_nodes < 2
+        error("Num supports must be greater than 2")
```

Comment:
This should say `num_nodes` and should specify we are talking about `FEGaussLobatto`